tests/jobs/localisation/example_case/test_full.py

```diff
@@ -0,0 +1,385 @@
+                },
+                "response": {
+                    "file_format": "ROFF",
+                    "grid_file_name": "GRID_STANDARD_UPSCALED.EGRID",
+                },
+            }
+        ),
+        pytest.param(
+            {
+                "case_name": "B",
+                "model_size": {
+                    "use_eclipse_grid_index_origo": True,
+                },
+                "field": {
+                    "file_format": "GRDECL",
+                    "grid_file_name": "GRID_STANDARD.EGRID",
+                },
+                "response": {
+                    "file_format": "GRDECL",
+                    "grid_file_name": "GRID_STANDARD_UPSCALED.EGRID",
+                },
+            }
+        ),
+        pytest.param(
+            {
+                "case_name": "C",
+                "model_size": {
+                    "use_eclipse_grid_index_origo": False,
+                },
+                "field": {
+                    "file_format": "ROFF",
+                    "grid_file_name": "GRID_RMS_ORIGO.EGRID",
+                },
+                "response": {
+                    "file_format": "ROFF",
+                    "grid_file_name": "GRID_RMS_ORIGO_UPSCALED.EGRID",
+                },
+            }
+        ),
+        pytest.param(
+            {
+                "case_name": "A2",
+                "model_size": {
+                    "use_eclipse_grid_index_origo": True,
+                    "polygon_file": "init_files/Polygons.txt",
+                },
+                "field": {
+                    "file_format": "ROFF",
+                    "grid_file_name": "GRID_WITH_ACTNUM.EGRID",
+                },
+                "response": {
+                    "file_format": "ROFF",
+                    "grid_file_name": "UpscaleGrid.EGRID",
+                },
+            }
+        ),
+        pytest.param(
+            {
+                "case_name": "D",
+                "model_size": {
+                    "use_eclipse_grid_index_origo": True,
+                },
+                "field": {
+                    "file_format": "ROFF",
+                    "grid_file_name": "GRID_STANDARD.EGRID",
+                },
+                "response": {
+                    "file_format": "ROFF",
+                    "grid_file_name": "GRID_STANDARD_UPSCALED.EGRID",
+                },
+                "localisation": {
+                    "method": "constant",
+                },
+            }
+        ),
+    ],
+)
+def test_that_localization_works_with_different_settings(
+    tmp_path: Path, monkeypatch: pytest.MonkeyPatch, new_settings: Dict[str, Any]
+):
+    monkeypatch.chdir(tmp_path)
+    print(f"Setup test case on tmp_path:  {tmp_path}")
+    settings = Settings()
+    settings.update(new_settings)
+
+    generate_seed_file(settings.field.seed_file)
+
+    # Make available data to generate grid
+    polygon_file = None
+    if settings.case_name == "A2":
```

Comment:
Can we avoid loading "Polygons.txt" and somehow re-create it?
Also, could we use a more descriptive case name than "A2"?